spec/features/food_search_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+RSpec.describe 'Welcome Page' do
+    describe 'when I search for a food in the search form' do
+        it 'can return a result of 10 foods from the relevant search ingredient' do
+            visit root_path
+            fill_in :q, with: 'sweet potatoes'
+            click_on "Search"
+            expect(current_path).to eq('/foods')
+
+            expect(page).to have_content("45635 Results")
+            expect(page).to have_content("492111402857")
+            expect(page).to have_content("SWEET POTATOES")
+            expect(page).to have_content("ARCHER FARMS")
+            expect(page).to have_content("SWEET POTATOES.")
+        end
+    end
+end
```

Comment:
You can also be a bit more specific in your tests, maybe add some classes in your views to target specific parts of your page that you'd expect specific content to be located.